aws-datastore/src/test/java/com/amplifyframework/datastore/DataStoreConfigurationTest.java

```diff
@@ -137,7 +139,7 @@ public void testInvalidKeyThrowsException() throws DataStoreException, JSONExcep
         public void onConflictDetected(
                 @NonNull ConflictData<? extends Model> conflictData,
                 @NonNull Consumer<ConflictResolutionDecision<? extends Model>> onDecision) {
-            onDecision.accept(ConflictResolutionDecision.retry(null));
+            onDecision.accept(ConflictResolutionDecision.retry(RandomString::string));
```

Comment:
Hm. Isn't this field supposed to store a model instance?

Reply:
Not really sure what it was, but my Android Studio was complaining that a `null` value is being supplied into `@NonNull` parameter, so I just put in a fake Model haha

Reply:
Well but it says `RandomString::string` -- not `SomeModel.instance()`, that was my point.